qt_dotgraph/CMakeLists.txt

```diff
@@ -7,5 +7,7 @@ catkin_package()
 catkin_python_setup()
 
 if (CATKIN_ENABLE_TESTING)
-  catkin_add_nosetests(test)
+  catkin_add_nosetests(test/dot_to_qt_test.py)
+  catkin_add_nosetests(test/pydot_factory_test.py)
+  catkin_add_nosetests(test/pygraphviz_factory_test.py)
```

Comment:
Since this change is not related to style cleanup please keep it in a separate PR. I also don't see why it should be necessary. If `nosetest` doesn't pick all of these tests up automatically maybe something else could be changed?

Reply:
Please revert this change and apply the fix described in https://github.com/ros-visualization/rqt/pull/140#discussion_r229806336.

Reply:
@mlautman Friendly ping.

Reply:
This was moved to pytests in PR #133 and PR #136

Reply:
The commits must have been rebased since then since this change doesn't exist in the finally merged patch anymore.